e2e-tests/smart-update/run

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+IMAGE_PXC=${IMAGE_PXC:-"percona/percona-xtradb-cluster-operator:1.4.0-pxc8.0"}
+IMAGE_TO_UPDATE=${IMAGE_TO_UPDATE:-"perconalab/percona-xtradb-cluster-operator:master-pxc8.0"}
+test_dir=$(realpath $(dirname $0))
+. ${test_dir}/../functions
+API="pxc.percona.com/v1-5-0"
```

Comment:
can we spin up Version service here? from the main branch and add v9.9.9 API version and IMAGE_PXC_TO_UPDATE  into it

```suggestion
test_dir=$(realpath $(dirname $0))
. ${test_dir}/../functions

API='pxc.percona.com/v9-9-9'
IMAGE_PXC_TO_UPDATE=${IMAGE_PXC}
PXC_VER=$(echo -n "${IMAGE_PXC_TO_UPDATE}" | $sed -r 's/.*([0-9].[0-9])$/\1/')
IMAGE_PXC='percona/percona-xtradb-cluster-operator:1.4.0-pxc$PXC_VER'
```

Reply:
done